mtc.go

```diff
@@ -2039,3 +2044,68 @@ func (tai *TrustAnchorIdentifier) unmarshal(s *cryptobyte.String) error {
 	tai.Issuer = oid
 	return nil
 }
+
+// VerifyOpts includes parameters for verifying a BikeshedCertificate.
+type VerifyOptions struct {
+	// SignedValidityWindow is a signed validity window that covers the
+	// certificate. It is the caller's responsibility to verify the
+	// validity window's signature using the CA parameters.
```

Comment:
> It is the caller's responsibility ...

This is a footgun, but it seems sensible since the same signed validity window will be used to validate many MTCs by the client. That said, there are things we can do to make the API safer.

Reply:
Why not pass in a `ValidityWindow` instead if we're not verifying the signature?

Reply:
That's a really good idea. Done.